src/System.Globalization.Calendars/tests/JapaneseCalendar/JapaneseCalendarToFourDigitYear.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Globalization.CalendarsTests
+{
+    public class JapaneseCalendarToFourDigitYear
+    {        
+        [Theory]
+        [InlineData(1)]
+        [InlineData(99)]
+        [InlineData(2016)]
+        public void ToFourDigitYear(int year)
+        {
+            Calendar calendar = new JapaneseCalendar();
```

Comment:
I suggest you set explicit TwoDigitYearMax value before checking the ToFourDigitYear. doing that will guarantee the result regardless of the running OS.